### PR TITLE
8.0 oca corrections crm claim type dev osval

### PR DESCRIPTION
**Pull request by yaniaular**
2nd part of the corrections made in https://github.com/OCA/crm/pull/39.

### CODE REVIEW

**Comment by yaniaular** [comment]
We will be working in this branch corrections made in https://github.com/OCA/crm/pull/39.

File: crm_claim_type/security/ir.model.access.csv [hunk 1]
```diff
@@ -1,3 +1,4 @@
 id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
-access_crm_claim_type_user,crm.claim.type.user,model_crm_claim_type,base.group_sale_manager,1,1,1,1
-
+access_crm_claim_type_technical,crm.claim.type.technical,model_crm_claim_type,base.group_no_one,1,1,1,1
+access_crm_claim_type_manager,crm.claim.type.manager,model_crm_claim_type,base.group_sale_manager,1,0,0,0
+access_crm_claim_type_user,crm.claim.type.user,model_crm_claim_type,base.group_sale_salesman,1,0,0,0
```
Comment (nhomar) [inline crm_claim_type/security/ir.model.access.csv:4]: Add a new line.

File: crm_claim_type/views/crm_case_claims_form_view.xml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+            <field name="model">crm.claim</field>
+            <field name="inherit_id" ref="crm_claim.crm_case_claims_form_view"/>
+            <field name="arch" type="xml">
+                <xpath expr="//field[@name='name']" position="after">
+                    <field name="claim_type" required="True" options="{'no_create': True}"/>
+                </xpath>
+            </field>
+        </record>
+    </data>
+</openerp>
```
Comment (nhomar) [inline crm_claim_type/views/crm_case_claims_form_view.xml:15]: Add a new line.

**Comment by nhomar** [comment]
The 2 comments can be done in other PR.

**Pull request merged**